modules/storages/app/components/storages/project_storages/projects/table_component.rb

```diff
@@ -0,0 +1,48 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2024 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+# Purpose: Defines a table based on TableComponent for listing the
+# Projects for a given Storage.
+# See also: row_component.rb, which contains a method
+# for every "column" defined below.
+module Storages::ProjectStorages::Projects
+  class TableComponent < Projects::TableComponent # rubocop:disable OpenProject/AddPreviewForViewComponent
+    include OpTurbo::Streamable
+
+    def sortable?
+      false
+    end
```

Comment:
Oddly enough this doesn't seem to kick in:

- [ ] I still see the "sorting icon" - which should be disabled in this feature.
- [ ] The action list is filled in with the default properties, in this feature spec- the menu actions are (1) Edit Project Folder (2) Remove project - I suggest either leaving them empty or defining placeholders.

![Screenshot 2024-07-05 at 11 15 57 AM](https://github.com/opf/openproject/assets/17295175/9233c431-9daa-4b45-8b92-4f96df029e99)

Reply:
https://github.com/opf/openproject/assets/17295175/b2a0d89b-4261-4aaf-b8b5-2925c13161f4